deployments/helm/README.md

```diff
@@ -0,0 +1,22 @@
+## Install KubeArmor using helm
+* Install kubearmor via helm, enable/disable kuberarmor relay by specifying either true or false.
+* Specify the namespace.
+* Specify environment depends on your environment like { docker, microk8s, minikube, k3s and generic (GKE, EKS)} by default it is generic.
+
+```
+helm upgrade --install kubearmor . \
+    --set kubearmorrelay.enabled=true \
+    --set environment.name=<environment> \
+    --set namespace.name=<namespace> \
+    -n <namespace>
```

Comment:
What is the use of `-n namespace` option? If I use this then it does not seem to be using the namespace provided .. whereas `--set namespace.name` works ok.
Can we remove -n option?